docs/content/tutorial/step_06.ngdoc

```diff
@@ -230,6 +230,9 @@ You can now rerun `npm run protractor` to see the tests run.
 
 * Reverse the sort order by adding a `-` symbol before the sorting value:
   `<option value="-age">Oldest</option>`
+  Upon refreshing the page, you'll see that the default selection of the "age" option in "Sort by" is blank again.
```

Comment:
I would rephrase this to something like:

> You'll notice that the default ordering by age does not work any more and the drop-down list has a blank option selected.
> Fix this, by updating the `orderProp` value in `phone-list.component.js` to match the new value on the `<option>` element.

Reply:
I updated the documentation to more closely follow your suggested change. Thank you for improving the clarity! Fixed in https://github.com/angular/angular.js/pull/16781/commits/41e28b65e4ad8d18fef7a74ff57d3944c4caa4c4